<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>EditorDailyStatisticsMetaInfo</class>
 <widget class="QWidget" name="EditorDailyStatisticsMetaInfo">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>871</width>
    <height>515</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>EditorDailyStatisticsMetaInfo</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QFormLayout" name="formLayout">
     <property name="labelAlignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
     <item row="0" column="0">
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Id:</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLineEdit" name="Id"/>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>Uuid:</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QLineEdit" name="Uuid"/>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>ApiVersion:</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QLineEdit" name="ApiVersion"/>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="label_4">
       <property name="text">
        <string>Name:</string>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QLineEdit" name="Name"/>
     </item>
     <item row="4" column="0">
      <widget class="QLabel" name="label_5">
       <property name="text">
        <string>Description:</string>
       </property>
      </widget>
     </item>
     <item row="4" column="1">
      <widget class="QLineEdit" name="Description"/>
     </item>
     <item row="5" column="0">
      <widget class="QLabel" name="label_6">
       <property name="text">
        <string>Day:</string>
       </property>
      </widget>
     </item>
     <item row="5" column="1">
      <widget class="QLineEdit" name="Day"/>
     </item>
     <item row="6" column="0">
      <widget class="QLabel" name="label_7">
       <property name="text">
        <string>Created Project Number:</string>
       </property>
      </widget>
     </item>
     <item row="6" column="1">
      <widget class="QLineEdit" name="CreatedProjectNumber">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item row="7" column="0">
      <widget class="QLabel" name="label_8">
       <property name="text">
        <string>Created Swc Number:</string>
       </property>
      </widget>
     </item>
     <item row="7" column="1">
      <widget class="QLineEdit" name="CreatedSwcNumber"/>
     </item>
     <item row="8" column="0">
      <widget class="QLabel" name="label_9">
       <property name="text">
        <string>Create Swc Node Number:</string>
       </property>
      </widget>
     </item>
     <item row="8" column="1">
      <widget class="QLineEdit" name="CreateSwcNodeNumber"/>
     </item>
     <item row="9" column="0">
      <widget class="QLabel" name="label_10">
       <property name="text">
        <string>Deleted Project Number:</string>
       </property>
      </widget>
     </item>
     <item row="9" column="1">
      <widget class="QLineEdit" name="DeletedProjectNumber"/>
     </item>
     <item row="10" column="0">
      <widget class="QLabel" name="label_11">
       <property name="text">
        <string>Deleted Swc Number:</string>
       </property>
      </widget>
     </item>
     <item row="10" column="1">
      <widget class="QLineEdit" name="DeletedSwcNumber"/>
     </item>
     <item row="11" column="0">
      <widget class="QLabel" name="label_12">
       <property name="text">
        <string>Delete Swc Node Number:</string>
       </property>
      </widget>
     </item>
     <item row="11" column="1">
      <widget class="QLineEdit" name="DeletedSwcNodeNumber"/>
     </item>
     <item row="12" column="0">
      <widget class="QLabel" name="label_13">
       <property name="text">
        <string>Modified Project Number:</string>
       </property>
      </widget>
     </item>
     <item row="12" column="1">
      <widget class="QLineEdit" name="ModifiedProjectNumber"/>
     </item>
     <item row="13" column="0">
      <widget class="QLabel" name="label_14">
       <property name="text">
        <string>Modified Swc Number:</string>
       </property>
      </widget>
     </item>
     <item row="13" column="1">
      <widget class="QLineEdit" name="ModifiedSwcNumber"/>
     </item>
     <item row="14" column="0">
      <widget class="QLabel" name="label_15">
       <property name="text">
        <string>Modified Swc Node Number:</string>
       </property>
      </widget>
     </item>
     <item row="14" column="1">
      <widget class="QLineEdit" name="ModifiedSwcNodeNumber"/>
     </item>
     <item row="15" column="0">
      <widget class="QLabel" name="label_16">
       <property name="text">
        <string>Project Query Number:</string>
       </property>
      </widget>
     </item>
     <item row="15" column="1">
      <widget class="QLineEdit" name="ProjectQueryNumber"/>
     </item>
     <item row="16" column="0">
      <widget class="QLabel" name="label_17">
       <property name="text">
        <string>Swc Query Number:</string>
       </property>
      </widget>
     </item>
     <item row="16" column="1">
      <widget class="QLineEdit" name="SwcQueryNumber"/>
     </item>
     <item row="17" column="0">
      <widget class="QLabel" name="label_18">
       <property name="text">
        <string>Node Query Number:</string>
       </property>
      </widget>
     </item>
     <item row="17" column="1">
      <widget class="QLineEdit" name="NodeQueryNumber"/>
     </item>
     <item row="18" column="0">
      <widget class="QLabel" name="label_19">
       <property name="text">
        <string>Active User Number:</string>
       </property>
      </widget>
     </item>
     <item row="18" column="1">
      <widget class="QLineEdit" name="ActiveUserNumber"/>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
